.github/workflows/build_docker_push_worker.yml

```
@@ -147,3 +147,80 @@ jobs:
+      WORKER_IMAGE_NAME: reearth/reearth-flow-worker:nightly
+      WORKER_IMAGE_NAME_GHCR: ghcr.io/reearth/reearth-flow/reearth-flow-worker:latest
+      WORKER_IMAGE_NAME_GCP: asia-northeast1-docker.pkg.dev/reearth-plateau-dev/reearth-plateau/reearth-flow-worker:latest
+    steps:
+      - uses: google-github-actions/auth@v2
+        with:
+          service_account: ${{ secrets.GCP_DEV_SERVICE_ACCOUNT }}
+          workload_identity_provider: ${{ secrets.GCP_DEV_WORKLOAD_IDENTITY_PROVIDER }}
+      - name: Configure docker
+        run: gcloud auth configure-docker asia-northeast1-docker.pkg.dev --quiet
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)